### DISCUSSION TITLE
You need a Galaxy Brain tag ??

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#1] [Q&A] [answered]
If you need the tag then answer in this discussion and I will mark yours as the answer.

#FCFS

**Comment by yaxan** [comment, marked as answer]
lmao I could use one

**Reply by Sriansh-raj** [reply]
lol you got this 😄

**Reply by yaxan** [reply]
weird I didn't get one

**Reply by Sriansh-raj** [reply]
Well you need to answer two :)

**Reply by Sriansh-raj** [reply]
@yaxan go to #18 for earn fr.

**Comment by bytesleo** [comment]
Hello! :)

**Reply by Sriansh-raj** [reply]
Hola @kevoj :octocat: 
Go to #18 .
:)
